react/src/views/Transfers/ShipmentsOverview/components/TableCells.tsx

```diff
@@ -43,10 +43,12 @@ export function StateCell({ value }: CellProps<any>) {
   let color = "inherit";
 
   // TODO: Receiving State is missing in type ShipmentState
-  if (value === ShipmentState.Preparing) {
+  if (value === ShipmentState.Preparing || value === ShipmentState.Receiving) {
```

Comment:
The todo comment above can now be removed

Reply:
done